crates/workspace/src/workspace_settings.rs

```diff
@@ -262,6 +262,15 @@ pub struct CenteredLayoutSettings {
     ///
     /// Default: 0.2
     pub right_padding: Option<f32>,
+    /// Whether to show the border around panes.
+    ///
+    /// Default: true
+    #[serde(default = "default_show_border")]
```

Comment:
```suggestion
    #[serde(default = "default_true")]
```

Let's reuse the existing method for this.